uwrt_mars_rover_hw/drivers/roboteq_driver/mbs/roboteq_control_mode_switching.mbs

```diff
@@ -0,0 +1,18 @@
+print("Starting roboteq drivetrain control mode switching ... \r")

```

Comment:
Im not sure if this matters to MBS but the [manual](https://www.roboteq.com/index.php/docman/motor-controllers-documents-and-files/documentation/user-manual/650-microbasic-scripting-manual/file) has all the functions as `PascalCase`